packages/utilities/fast-react-wrapper/src/index.ts

```diff
@@ -101,7 +102,7 @@ function getTagName<TElement, TEvents>(
         const definition = FASTElementDefinition.getByType(type)!;
 
         if (definition) {
-            config.name = definition.name;
+            return definition.name;
```

Comment:
@EisenbergEffect - @nicholasrice and I noticed the mutation of config here while exploring the test failures related to #6747. Curious if there are issues with this approach which essentially returns name as a constant and utilizes it everywhere in place of the other direct references to `config.name` to ensure consistency.